db/migrate/20191023204745_update_test_status_for_discounted_drivers.rb

```diff
@@ -0,0 +1,18 @@
+class UpdateTestStatusForDiscountedDrivers < ActiveRecord::Migration[5.1]
+  def up
+    SupervisorReport.all.each do |report|
+      if(report.driver_discounted)
```

Comment:
In general, shouldn't need parentheses around conditionals in Ruby. Applies to all of the PR content.